spyder/plugins/ipythonconsole.py

```diff
@@ -1390,6 +1390,18 @@ def process_finished(self, client):
         if self.variableexplorer is not None:
             self.variableexplorer.remove_shellwidget(id(client.shellwidget))
 
+    def connect_external_kernel_to_varexp(self, shellwidget):
+        """Connect a shellwidget to the variable explorer."""

```

Comment:
This needs to be

    Connect an external kernel to the Variable Explorer and Help.